src/audio/host.c

```diff
@@ -246,9 +246,11 @@ static uint32_t host_get_copy_bytes_one_shot(struct comp_dev *dev)
 
 	/* calculate minimum size to copy */
 	if (dev->direction == SOF_IPC_STREAM_PLAYBACK)
-		copy_bytes = hd->local_buffer->stream.free;
+		copy_bytes = audio_stream_get_free_bytes(
+			&hd->local_buffer->stream);
 	else
```

Comment:
You can have lines up to 100 chars, keep this in mind. Not sure if this goes above the limit.

Reply:
Yikes my code still tries to limit itself to 80. Though if I copied this portion in particular, it wouldn't even go above the 80 chars limit? Are there only two tabs? (with two tabs it's 71 characters).

Reply:
cleaned up